bot/exts/evergreen/youtube.py

```diff
@@ -0,0 +1,77 @@
+from typing import Dict, List, Optional
+
+from discord import Embed
+from discord.ext.commands import Bot, Cog, Context, command, cooldown
```

Comment:
The prevailing style for Sir Lancebot is to do `from discord.ext import commands` and then use the commands namespace for everything EG `commands.Bot`

Reply:
Sounds good 👍